tools/lsp/properties.rs

```diff
@@ -544,9 +543,7 @@ fn get_element_information(
     ElementInformation {
         id: e.id.clone(),
         type_name: format!("{}", e.base_type),
```

Comment:
```suggestion
        type_name: e.base_type.to_string(),
```

(just some style thing)